src/test/java/org/dinosaur/foodbowl/domain/review/presentation/ReviewControllerTest.java

```diff
@@ -0,0 +1,424 @@
+import org.dinosaur.foodbowl.domain.member.domain.vo.SocialType;
+import org.dinosaur.foodbowl.domain.review.application.ReviewService;
+import org.dinosaur.foodbowl.domain.review.dto.request.ReviewCreateRequest;
+import org.dinosaur.foodbowl.file.FileTestUtils;
+import org.dinosaur.foodbowl.test.PresentationTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.web.multipart.MaxUploadSizeExceededException;
+
+@SuppressWarnings("NonAsciiCharacters")
+@WebMvcTest(controllers = ReviewController.class)
+class ReviewControllerTest extends PresentationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private JwtTokenProvider jwtTokenProvider;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private ReviewService reviewService;
+
+    @BeforeEach
+    void setUp() {
+        Member member = Member.builder()
+                .email("foodBowl@gmail.com")
+                .socialId("foodBowlId")
+                .socialType(SocialType.APPLE)
+                .nickname("foodbowl")
+                .introduction("푸드볼 서비스")
+                .build();
+        given(memberRepository.findById(anyLong()))
+                .willReturn(Optional.of(member));
+    }
+
+    @Nested
+    class 리뷰_작성_시_ {
+
+        private final String accessToken = jwtTokenProvider.createAccessToken(1L, ROLE_회원);
+
+        @Test
+        void 사진이_포함된_경우_200_상태코드를_반환한다() throws Exception {
+            given(reviewService.create(any(ReviewCreateRequest.class), anyList(), any(Member.class)))
+                    .willReturn(1L);
+            ReviewCreateRequest reviewCreateRequest = generateReviewCreateDto();
+            MockMultipartFile request = new MockMultipartFile("request", "", "application/json",
+                    objectMapper.writeValueAsBytes(reviewCreateRequest));
+            MockMultipartFile multipartFile1 = (MockMultipartFile) FileTestUtils.generateMockMultiPartFile();
+            MockMultipartFile multipartFile2 = (MockMultipartFile) FileTestUtils.generateMockMultiPartFile();
+
+            mockMvc.perform(multipart("/v1/reviews")
+                            .file(request)
+                            .file(multipartFile1)
+                            .file(multipartFile2)
+                            .header("Authorization", "Bearer " + accessToken)
+                            .characterEncoding(StandardCharsets.UTF_8))
```

Comment:
```suggestion
                            .file(request)
                            .file(multipartFile1)
                            .file(multipartFile2)
                            .header("Authorization", "Bearer " + accessToken)
                            .characterEncoding(StandardCharsets.UTF_8))
```
```suggestion
                            .header("Authorization", "Bearer " + accessToken)
                            .characterEncoding(StandardCharsets.UTF_8)
                            .file(request)
                            .file(multipartFile1)
                            .file(multipartFile2))
```
요것도 사소하지만 헤더를 위로 올리는 건 어떠신가요 :)

Reply:
요것도 정해지면 테스트 전반적으로 확인 부탁드리겠습니다 ✅

Reply:
![스크린샷 2023-08-23 오후 11 03 42](https://github.com/Busan-Dinosaur/Foodbowl-Backend/assets/68818952/0e22ff1a-aa85-4685-b089-111db61e5e0e)

체이닝하는 대상이 바뀌어서 file이 먼저 위치해야 합니다 ㅎㅎ

Reply:
`multipart` 요청을 많이 사용해보지 않았는데, 체이닝하는 대상이 달라지네요 🤓 하나 배워갑니다~~